smoke-tests/apps/MicrometerOldVersion/build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+plugins {
```

Comment:
Can we rename this smoke test to be more specific, something like "MicrometerPrior1_5"